src/App.tsx

```diff
@@ -3,12 +3,28 @@ import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
-type State = {};
+type State = {
+  query: string;
+};
 
 export class App extends React.Component<{}, State> {
-  state: State = {};
+  state: State = {
+    query: '',
+  };
+
+  getFilteredMovies = () => {
+    const { query } = this.state;
+    const filteredMovies = moviesFromServer.filter(movie => (
+      (movie.title + movie.description).toLocaleLowerCase()
```

Comment:
```suggestion
      (movie.title + ' ' + movie.description).toLocaleLowerCase()
```
![image](https://user-images.githubusercontent.com/64437769/133052047-c3616a65-b0d0-413a-9693-6db356a410fc.png)